modules/siddhi-core/src/main/java/io/siddhi/core/table/Table.java

```diff
@@ -99,38 +103,44 @@ public void initTable(TableDefinition tableDefinition, StreamEventFactory storeE
         this.siddhiAppContext = siddhiAppContext;
         this.recordTableHandler = recordTableHandler;
         Element onErrorElement = getAnnotationElement(ANNOTATION_STORE, ANNOTATION_ELEMENT_ON_ERROR,
-                tableDefinition.getAnnotations());
+            tableDefinition.getAnnotations());
         if (onErrorElement != null) {
             this.onErrorAction = OnErrorAction.valueOf(onErrorElement.getValue());
         }
+        if (this.onErrorAction == OnErrorAction.STORE && siddhiAppContext.getSiddhiContext().getErrorStore() == null) {
+            // TODO: 2020-10-02 add in streams
+            LOG.error("On error action is 'STORE' for table " + tableDefinition.getId() + " in Siddhi App "
+                    + siddhiAppContext.getName() + " but error store is not configured in Siddhi Manager");
+        }
+        this.isObjectColumnPresent = isObjectColumnPresent(tableDefinition);
         if (siddhiAppContext.getStatisticsManager() != null) {
             latencyTrackerFind = QueryParserHelper.createLatencyTracker(siddhiAppContext, tableDefinition.getId(),
-                    SiddhiConstants.METRIC_INFIX_TABLES, SiddhiConstants.METRIC_TYPE_FIND);
+                SiddhiConstants.METRIC_INFIX_TABLES, SiddhiConstants.METRIC_TYPE_FIND);
```

Comment:
Please check the formatting here and in other similar places as well

Reply:
Fixed in a646550